### DISCUSSION TITLE
Allow Date Range Picker to select same date for start and end

### DISCUSSION THREAD

**Discussion by giray123** [mantinedev/mantine#4027] [Feature requests]
Current Date Range Picker does not allow to select the same day as the start and the end date of the range. Upon clicking on the same date twic, it deselects. Is it possible to change the behaviour accordingly?

**Comment by rtivital** [comment]
You can do that by setting `allowSingleDateInRange` prop, it is supported by all components that have dates range selection option. See documentation (v6) – https://mantine.dev/dates/date-picker/#single-date-in-range
In v5 and earlier versions, the same prop (`allowSingleDateInRange`) is available on DateRangePicker component.

**Reply by giray123** [reply]
Thank you very much, I have missed it somehow. I appreciate the attention to details. 👍
